data/sql/updates/pending_db_world/mad_servant.sql

```diff
@@ -0,0 +1,13 @@
+--
+INSERT INTO `creature_text` (`CreatureID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES ('15111', 'I gonna make you into mojo!', '12', '100', '10435');
+INSERT INTO `creature_text` (`CreatureID`, `ID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES ('15111', '1', 'Troll mojo da strongest mojo!', '12', '100', '10437');
```

Comment:
```suggestion
DELETE FROM `creature_text` WHERE `CreatureID`=15111 AND `GroupID`=0;
INSERT INTO `creature_text` (`CreatureID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES (15111, 'I gonna make you into mojo!', 12, 100, 10435);
INSERT INTO `creature_text` (`CreatureID`, `ID`, `Text`, `Type`, `Probability`, `BroadcastTextId`) VALUES (15111, 1, 'Troll mojo da strongest mojo!', 12, 100, 10437);
```

Reply:
First, remember that only text needs to include ``    |    '15111' -> 15111
Also remember that before every INSERT there should be a DELETE, in order to avoid this error:

![image](https://user-images.githubusercontent.com/61223313/173261915-8afd8634-4b4e-4483-bf02-d1c69c16a836.png)

Reply:
Ok, but even if there is no entry to begin with? The delete won't do anything, is that just a SQL formality/convention? I didn't get any errors when testing.

Reply:
It's part of best practices of AzerothCore to always add a delete before an insert to make them reusable SQL queries. You should be able to always get a successful operation when running SQL updates.

Also if the original sql version of yours were added to the AC updates, when performing DB updates in the following weeks the error may have come up if anyone runs updates twice.

Reply:
AH now I see it. I haven't thought about it. It doesn't allow overwriting if I don't put in the DELETE beforehand. Keira does this automatically.
Thanks for pointing that out, it went right over my head.